[2022-04-24 22:52:09,056] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.download_crash_dataset_task manual__2022-04-24T22:27:24.245486+00:00 [queued]>
[2022-04-24 22:52:09,169] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: local_2_gcs_2_bq_dag.download_crash_dataset_task manual__2022-04-24T22:27:24.245486+00:00 [queued]>
[2022-04-24 22:52:09,179] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 22:52:09,180] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2022-04-24 22:52:09,182] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-24 22:52:09,260] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_crash_dataset_task> on 2022-04-24 22:27:24.245486+00:00
[2022-04-24 22:52:09,278] {standard_task_runner.py:52} INFO - Started process 1490 to run task
[2022-04-24 22:52:09,338] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'local_2_gcs_2_bq_dag', 'download_crash_dataset_task', 'manual__2022-04-24T22:27:24.245486+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/download_2_gcs_dag_V3.py', '--cfg-path', '/tmp/tmpr_c5yy2h', '--error-file', '/tmp/tmpp5txdkp0']
[2022-04-24 22:52:09,347] {standard_task_runner.py:77} INFO - Job 100: Subtask download_crash_dataset_task
[2022-04-24 22:52:09,896] {logging_mixin.py:109} INFO - Running <TaskInstance: local_2_gcs_2_bq_dag.download_crash_dataset_task manual__2022-04-24T22:27:24.245486+00:00 [running]> on host 2bdc5459fc64
[2022-04-24 22:52:10,252] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_2_gcs_2_bq_dag
AIRFLOW_CTX_TASK_ID=download_crash_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-24T22:27:24.245486+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-24T22:27:24.245486+00:00
[2022-04-24 22:52:10,255] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-24 22:52:10,260] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://data.cityofchicago.org/api/views/85ca-t3if/rows.csv?accessType=DOWNLOAD > /opt/***/85ca-t3if.csv']
[2022-04-24 22:52:10,355] {subprocess.py:85} INFO - Output:
[2022-04-24 22:53:10,298] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-24 22:53:10,368] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=local_2_gcs_2_bq_dag, task_id=download_crash_dataset_task, execution_date=20220424T222724, start_date=20220424T225209, end_date=20220424T225310
[2022-04-24 22:53:10,446] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-24 22:53:10,559] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
